src/repositories/ArtistRepository.ts

```diff
@@ -19,8 +19,15 @@ export class ArtistRepository {
     return await this.repository.findOneBy({ id });
   }
 
-  async findAll(): Promise<ArtistEntity[]> {
-    return await this.repository.find();
+  async findAll(options?: { skip?: number; take?: number }): Promise<ArtistEntity[]> {
+    const { skip = 0, take = 10 } = options ?? {};
+
+    return await this.repository.find({
+      skip,
+      take,
+      select: ['id', 'name', 'genre', 'bio'],
+      order: { createdAt: 'DESC' }
+    });
   }
```

Comment:
- You should only remove password. Return the rest.
- Return the first 100 records. 10 is too small.
```suggestion
  async findAll(options?: { skip?: number; take?: number }): Promise<Omit<ArtistEntity, 'password'>[]> {
    const { skip = 0, take = 10 } = options ?? {};

    const artists = await this.repository.find({
      skip,
      take,
      order: { createdAt: 'DESC' }
    });

    return artists.map(({ password, ...artist }) => artist);
  }
```